README.md

```diff
@@ -80,6 +80,8 @@ The architecture has been designed according to the **Atomic Design approach**.
 
 #### Other:
 - Code Review in every Pull Request (and 2 Approve to merge)
+- Event Storming & Event Modeling
```

Comment:
Nie jestem pewien czy to jest dobre miejsce w **technologiach**. :) ?

Reply:
Przyznam się, że nie wiem gdzie o tym napisać. Uznałem, że to wymieniane są inne rzeczy niż te tylko do fronta lub tylko do backu i tu wrzuciłem. 
Teraz sobie myślę, że nie pogoglowac jutro jak wrócę od rodziny i gdzieś indziej to wstawić, choć nie wiem czy coś lepszego znajdę.